modules/ossm-rn-fixed-issues.adoc

```diff
@@ -19,10 +19,14 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {SMProductShortName} fixed issues
 
+* https://issues.redhat.com/browse/OSSM-3993[OSSM-3993] Previously, Kiali only supported OpenShift OAuth via a proxy on the standard HTTPS port of `443`. Now, Kiali supports it over a non-standard HTTPS port. To enable it, you must define `spec.server.web_port` in the Kiali CR to the proxy's non-standard HTTPS port.
```

Comment:
As per this link https://redhat-documentation.github.io/supplementary-style-guide/#release-notes-doc-texts, i would suggest to use before this update and with this update format, instead of using the current format.

Please apply this change globally for this particular section.

```suggestion
* https://issues.redhat.com/browse/OSSM-3993[OSSM-3993] Before this update, Kiali only supported OpenShift OAuth via a proxy on the standard HTTPS port of `443`. With this update, Kiali supports OpenShift OAuth over a non-standard HTTPS port. To enable the port, you must set the `spec.server.web_port` field to the proxy's non-standard HTTPS port in the Kiali CR.
```